bedrock/firefox/templates/firefox/welcome/page10.html

```diff
@@ -12,52 +10,82 @@
 
 {% block body_class %}welcome-page10{% endblock %}
 
+{% block experiments %}
+  {% if switch('welcome-10-experiment', ['en-US', 'en-CA']) %}
+    {{ js_bundle('firefox_welcome_page10_experiment') }}
+  {% endif %}
+{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_welcome_page10') }}
+{% endblock %}
+
 {% set _entrypoint = 'mozilla.org-firefox-welcome-10' %}
 {% set _utm_campaign = 'welcome-10-vpn' %}
-{% set _source = '?source=welcome10' %}
+
+{% if variation == '1' %}
+  {% set title = 'Get a <strong>secure connection</strong> on every device'|safe %}
+  {% set desc = 'Protect your internet connection — on your computer, your tablet and even your phone — with this limited-time offer.' %}
+  {% set source = '?source=welcome10-exp-v1' %}
+  {% set cta_text = 'Get Mozilla VPN' %}
+{% elif variation == '2' %}
+  {% set title = 'Blur your location online for <strong>next-level privacy</strong>'|safe %}
+  {% set desc = 'Get Mozilla VPN for ' + vpn_monthly_price(plan='12-month', lang=LANG) + ' — this limited-time offer ends soon.' %}
```

Comment:
You're not using any geo-location here, so I don't think `vpn_monthly_price(plan='12-month', lang=LANG)` is really needed and you could straight up hard code `US$4.99/month`?

This is also using the 12-month plan price, which isn't yet available in the US (even though the monthly monetary value is the same).